documentation/modules/ROOT/pages/integrations/tracing.adoc

```diff
@@ -9,36 +9,103 @@
+* https://mvnrepository.com/artifact/io.opentelemetry.instrumentation/opentelemetry-kafka-clients-2.6/[opentelemetry-kafka-clients-2.6]
+* https://mvnrepository.com/artifact/io.opentelemetry.instrumentation/opentelemetry-kafka-clients-common/[opentelemetry-kafka-clients-common]
+ +
+To ensure version compatibility among the different OpenTelemetry packages, use Maven to download the packages.
+
+2. Create a POM file similar to the one in the following example.
+In your version of the file, specify the version of the OpenTelemetry package that you want to use.
+
+[source,xml]
+----
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+ <modelVersion>4.0.0</modelVersion>
+ <groupId>group.id</groupId>
+ <artifactId>artifact-id</artifactId>
+ <version>0.0.1</version>
+ <properties>
+  <version.opentelemetry.api>1.23.1</version.opentelemetry.api>
+  <version.opentelemetry.kafka.clients.2.6>1.23.0-alpha</version.opentelemetry.kafka.clients.2.6>
+ </properties>
+ <dependencies>
+  <dependency>
+   <groupId>io.opentelemetry</groupId>
+   <artifactId>opentelemetry-api</artifactId>
+   <version>${version.opentelemetry.api}</version>
+  </dependency>
+  <dependency>
+   <groupId>io.opentelemetry.instrumentation</groupId>
+   <artifactId>opentelemetry-kafka-clients-2.6</artifactId>
+   <version>${version.opentelemetry.kafka.clients.2.6}</version>
+  </dependency>
+ </dependencies>
+</project>
+----
+
+3. In the directory that contains the `pom.xml` file, run the following command to download all dependencies (including transitives) to a specified directory (for example. `./lib`)
 
-The tracing support is added to Debezium through the https://opentracing.io/[OpenTracing] specification.
-It is also necessary to provide a client implementing the specification.
-Debezium was tested with https://www.jaegertracing.io/[Jaeger] implementation.
+[source,bash]
+----
+mvn dependency:copy-dependencies -DoutputDirectory=./lib
+----
 
-[WARNING]
+==== Installing the SDK
+
+Install the OpenTelemetry SDK on your Kafka connect cluster by using the https://opentelemetry.io/docs/instrumentation/java/automatic/[Java agent] for automatic instrumentation.
+
+=== Use the {prodname} Kafka Connect image
+
+The OpenTelemetry API dependencies are installed in the https://quay.io/repository/debezium/connect[{prodname} Kafka Connect] container image.
+The JAR files are downloaded to a separate directory.
+By default, the JAR files are not added to the class path.
+To add the JAR files to the class path, set the environment variable `ENABLE_OTEL` to `true`.
+You must also install the https://opentelemetry.io/docs/instrumentation/java/automatic/[OpenTelemetry Java agent].
+
+=== Use Strimzi
+
+You can use the https://strimzi.io/[Strimzi] Kafka image, which provides all of the necessary OpenTelemetry components.
```

Comment:
If this is an alternative method for providing the required components, let's use this opening sentence to explain that, for example, something like "Instead of using the {prodname} Kafka Connect image as described in the previous section, you can use obtain all of the OpenTelemetry components by deploying the https://strimzi.io/[Strimzi]) Kafka image."

Reply:
see https://github.com/debezium/debezium/pull/4774#discussion_r1338854488